lms/templates/register.html

```diff
@@ -1,6 +1,7 @@
 <%inherit file="main.html" />
 <%namespace name='static' file='static_content.html'/>
 <%namespace file='main.html' import="login_query"/>
+<%namespace file="./util/gymcms.mako" />
```

Comment:
You're missing the `name='gymcms'` part of the import. 

But, the same comment from login.html applies here. This template is never being rendered because you're using the Logistration page. Keep the changes, revert the changes, or nuke the file all together. It doesn't make much of a difference.

Reply:
👋 bye template!